### PR TITLE
2380: edit court-issued docket entry

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: shared/src/business/useCases/docketEntry/fileCourtIssuedDocketEntryInteractor.js [hunk 1]
```diff
@@ -40,9 +40,9 @@ exports.fileCourtIssuedDocketEntryInteractor = async ({
 
   const caseEntity = new Case(caseToUpdate, { applicationContext });
 
-  const document = caseEntity.documents.find(
-    document => document.documentId === documentId,
-  );
+  const document = caseEntity.getDocumentById({
```
Comment (matthopson) [inline shared/src/business/useCases/docketEntry/fileCourtIssuedDocketEntryInteractor.js:43]: 👏 Thank you.

File: shared/src/proxies/documents/updateCourtIssuedDocketEntryProxy.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const { put } = require('../requests');
+
+/**
+ * updateCourtIssuedDocketEntryProxy
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {object} providers.document the document data
```
Comment (matthopson) [inline shared/src/proxies/documents/updateCourtIssuedDocketEntryProxy.js]: `documentMeta`

Comment (matthopson) [reply shared/src/proxies/documents/updateCourtIssuedDocketEntryProxy.js]: I'm guessing maybe this didn't get updated in the `file...Proxy` originally (which would be my bad).

Comment (rachaelparris) [reply shared/src/proxies/documents/updateCourtIssuedDocketEntryProxy.js]: 👍 I'll take care of that one too.

**Review by matthopson — APPROVED** [review]

File: web-api/serverless-case-documents.yml [hunk 3]
```diff
@@ -283,6 +283,18 @@ functions:
             authorizerId:
               Ref: ApiGatewayAuthorizer
 
+  updateCourtIssuedDocketEntry:
+    handler: web-api/src/caseDocumentsHandlers.updateCourtIssuedDocketEntryLambda
```
Comment (codyseibert) [inline web-api/serverless-case-documents.yml]: this should be `web-api/${self:provider.dir}/` instead of `web-api/src`

Comment (matthopson) [reply web-api/serverless-case-documents.yml]: Ah, probably broken in the other endpoint I made too. `court-issued-docket-entry`

**Review by kkoskelin — APPROVED** [review]

File: web-api/serverless-case-documents.yml (hunk 3 above)
Comment (codyseibert) [reply web-api/serverless-case-documents.yml]: I update them now

**Pull request merged**